src/scripts/main.js

```diff
@@ -1403,11 +1403,34 @@
 
         // TODO: Apply any global hunt details (such as from ongoing events, auras, etc).
         [
+            addEggHuntDetails,
             addLNYHuntDetails,
             addLuckyCatchHuntDetails
         ].forEach(details_func => details_func(message, user, user_post, hunt));
     }
 
+    /**
+     * Record the Eggscavator Charge level, both before and after the hunt.
+     * @param {Object <string, any>} message The message to be sent.
+     * @param {Object <string, any>} user The user state object, when the hunt was invoked (pre-hunt).
+     * @param {Object <string, any>} user_post The user state object, after the hunt.
+     * @param {Object <string, any>} hunt The journal entry corresponding to the active hunt.
+     */
+    function addEggHuntDetails(message, user, user_post, hunt) {
+        const quest = getActiveSEHQuest(user.quests);
+        if (quest) {
+            message.hunt_details = Object.assign(message.hunt_details || {}, {
+                is_egg_hunt: true,
+                egg_charge_pre: parseInt(quest.charge_quantity, 10),
+                can_double_eggs: (quest.charge_doubler === "active"),
+            });
+            const post_quest = getActiveSEHQuest(user_post.quests);
```

Comment:
Perhaps move the `post_quest` check up a few lines so that details are only assigned when both pre/post are available?